### PR TITLE
added storybook controls for confirm send ether component

### LINKED ISSUE #13072: ConfirmSendEther story: convert knobs and actions to controls / args
**Describe the bug**
As part of a push to convert all stories that use deprecated `@storybook/addon-knobs` and `@storybook/addon-actions` to use [controls](https://storybook.js.org/docs/react/essentials/controls) and [action argType annotation](https://storybook.js.org/docs/react/essentials/actions#action-argtype-annotation)

Convert `ConfirmSendEther` story in `ui/pages/confirm-send-ether/confirm-send-ether.stories.js` to use [controls](https://storybook.js.org/docs/react/essentials/controls) and [action argType annotation](https://storybook.js.org/docs/react/essentials/actions#action-argtype-annotation)

Read the [MetaMask extension storybook documentation guidelines](https://metamask.github.io/metamask-storybook/?path=/story/getting-started-documentation-guidelines--page) to get a better understanding of writing stories. Providing MDX documentation although helpful, is a bonus and not a requirement for this issue.

To start storybook:
```
yarn storybook
```

**Requirements**
- [ ] Story has `argTypes` that align with component api / props
- [ ] All instances of `@storybook/addon-knobs` have been removed in favour of control args
- [ ] All instances of `@storybook/addon-actions` have been removed in favour of [action argType annotation](https://storybook.js.org/docs/react/essentials/actions#action-argtype-annotation)

Any further improvements to the story interaction or documentation is a bonus

**Images**

Screen Shot 2021-12-12 at 6 21 30 PM

### PR DESCRIPTION
**Pull request by NidhiKJha**
* Fixes #13072

### Before

Screenshot 2022-07-14 at 7 14 05 PM

### After

Screenshot 2022-07-14 at 7 12 29 PM

### CODE REVIEW

File: ui/pages/confirm-send-ether/confirm-send-ether.stories.js [hunk 1]
```diff
@@ -1,59 +1,42 @@
...
 
+const options = [];
+const state = store.getState();
+const { identities } = state.metamask;
+Object.keys(identities).forEach(function (key) {
+  options.push({
+    label: identities[key].name,
+    address: key,
+  });
+});
```
Comment (georgewrmarshall) [inline ui/pages/confirm-send-ether/confirm-send-ether.stories.js:13]: Doesn't look like this is being used anywhere can we just remove it?

File: ui/pages/confirm-send-ether/confirm-send-ether.stories.js [hunk 2]
```diff
@@ -1,59 +1,42 @@
...
+      },
+      options: ['Receiver', '0xaD6D458402F60fD3Bd25163575031ACDce07538D'],
+    },
+    sender: {
+      control: {
+        type: 'select',
+      },
+      options: ['Sender'],
+    },
+  },
```
Comment (georgewrmarshall) [inline ui/pages/confirm-send-ether/confirm-send-ether.stories.js:31]: These don't look like the correct props? I think we can just remove.

```suggestion
```

File: ui/pages/confirm-send-ether/confirm-send-ether.stories.js [hunk 3]
```diff
@@ -1,59 +1,42 @@
-import React, { useEffect } from 'react';
-
-import { select } from '@storybook/addon-knobs';
+import React from 'react';
 import { store } from '../../../.storybook/preview';
```
Comment (georgewrmarshall) [inline ui/pages/confirm-send-ether/confirm-send-ether.stories.js:2]: Doesn't look like this is used? Can we remove?

```suggestion
```

File: ui/pages/confirm-send-ether/confirm-send-ether.stories.js (hunk 1 above)
Comment (georgewrmarshall) [inline ui/pages/confirm-send-ether/confirm-send-ether.stories.js:13]: Doesn't look like this is used? Can we remove?

```suggestion
```

File: ui/pages/confirm-send-ether/confirm-send-ether.stories.js [hunk 4]
```diff
@@ -1,59 +1,42 @@
...
+export const DefaultStory = (args) => {
+  return <ConfirmSendEther {...args} />;
 };
 
 DefaultStory.storyName = 'Default';
+
+DefaultStory.args = {
+  receiver: '0xaD6D458402F60fD3Bd25163575031ACDce07538D',
+  sender: 'Sender',
+};
```
Comment (georgewrmarshall) [inline ui/pages/confirm-send-ether/confirm-send-ether.stories.js:42]: Doesn't look like this is used can we remove?

```suggestion
```

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Looks like we can clean this story up a little. I know the issue just says to convert the story from knobs to controls but we can probably clean it up a bit. Remove all the unnecessary code and let's merge haha. No need to add any other argtypes. It displays without them.

**Review by georgewrmarshall — DISMISSED** [review]

**Review by kumavis — DISMISSED** [review]

**Review by brad-decker — APPROVED** [review]

**Review by georgewrmarshall — APPROVED** [review]

**Pull request merged**